backend/app/api/endpoints/openapi_responses.py

```
@@ -324,6 +344,249 @@ async def create_response(
+                        task_crd.status.updatedAt = datetime.now()
+                        task_resource.json = task_crd.model_dump(mode="json")
+                        task_resource.updated_at = datetime.now()
+                        from sqlalchemy.orm.attributes import flag_modified
+
+                        flag_modified(task_resource, "json")
+                        error_db.commit()
+            finally:
+                error_db.close()
+
+    # Start background task (fire-and-forget)
+    asyncio.create_task(_run_background_task())
```

Ruff (0.15.5)

[warning] Store a reference to the return value of `asyncio.create_task`

(RUF006)